### DISCUSSION TITLE
Should there be a way to indicate if a pet is a Service Animal

### DISCUSSION THREAD

**Discussion by JJ-Foust** [HACC2021/MVP#17] [Q&A] [answered]
This is important because it would affect the pickup time and would also affect others on the wait list since service animals get preference.

**Comment by JJ-Foust** [comment]
By having this information then automation could be setup to to move that animal up the waitlist for pickup when it arrives.

**Reply by judyn808** [reply]
I think this is a good idea because it adds a lot of user value. However, right now during COVID times, service animals are not being prioritized. Since services animals are more rare, I can imagine that an AQS employee could take it upon themselves to go the extra mile and flag the pet themselves (informally) and recall the owner asap. Phase 2 or 3?

**Comment by JJ-Foust** [comment, marked as answer]
They are not wanting to change their process at this time.
